lib/containers/common.pm

```diff
@@ -75,9 +85,7 @@ sub install_docker_when_needed {
                 assert_script_run "apt-get -y install docker-ce", timeout => 260;
             } else {
                 # We may run openSUSE with DISTRI=sle and openSUSE does not have SUSEConnect
-                if (can_build_sle_base && script_run("SUSEConnect --status-text | grep Containers", timeout => 240) != 0) {
-                    is_sle('<15') ? add_suseconnect_product("sle-module-containers", 12) : add_suseconnect_product("sle-module-containers");
-                }
+                activate_containers_module if (can_build_sle_base);
```

Comment:
I didn't want to remove this condition to not alter the logic. I think we should actually remove this condition since we are already making sure that we run SUSEConnect only when the host is sles.

Reply:
We do this if both the host and the container are SLE based.

Reply:
I don't understand. Maybe we can speak about it.

Reply:
So, in this file, we have 3 places where we need to activate the Container module 
1) install_podman_when_needed()
2) install_docker_when_needed()
3) install_buildah_when_needed()

But we ONLY use the condition `can_build_sle_base` in `install_docker_when_needed()`. I didn't want to change that condition in this PR, but I wonder why we have that condition there only and not in the other 2 functions.